pkg/cmd/init/init.go

```diff
@@ -61,5 +74,36 @@ func NewCmdInit() *cobra.Command {
 	cmd.Flags().StringVar(
 		&o.CustomParamsJSON, "custom-params", "",
 		i18n.T("Custom params in JSON string; if not empty, kusion will skip prompt process and use it as template default value"))
+
+	templatesCmd := newCmdTemplates()
+	cmd.AddCommand(templatesCmd)
+	return cmd
+}
+
+func newCmdTemplates() *cobra.Command {
+	o := NewTemplatesOptions()
+	cmd := &cobra.Command{
+		Use:                   "templates",
```

Comment:
the subcommand `kusion init templates` is also a little confusing, a subcommand like `kusion init list-templates` sounds more clear

Reply:
how about `kusion templates`, strip `template` cmd out of `init`